<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminGUI</class>
 <widget class="QWidget" name="AdminGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminGUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="genreLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Genre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="genreLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="yearLaber">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Year</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yearLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="nrLikesLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Number of Likes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nrLikesLine">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="trailerLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Trailer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="trailerLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="filterLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterLine"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="movieList"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="updateButton">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="undoButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="redoButton">
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="openTableButton">
         <property name="text">
          <string>Open TableView</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
